jiant/proj/main/components/container_setup.py

```diff
@@ -59,6 +59,16 @@ class JiantTaskContainer:
 
 
 def create_task_dict(task_config_dict: dict, verbose: bool = True) -> Dict[str, tasks.Task]:
+    """Make map of task name to task instances from map of task name to task config file paths.
+
+    Args:
+        task_config_dict (Dict): map from task name to task config filepath.
+        verbose (bool): True to print task config info.
+
+    Returns:
+        Dict mapping from task name to task instance.
+
+    """
     task_dict = {
         task_name: tasks.create_task_from_config_path(config_path=task_config_path, verbose=False)
```

Comment:
not related to the review but seems like `verbose=False` overwrites `verbose` here